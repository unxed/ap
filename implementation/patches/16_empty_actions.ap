version: "1.0"
changes:
  - file_path: "16_empty_actions.txt"
    modifications:
      - action: REPLACE
        target:
          snippet: "Line 1 to be replaced with nothing."
        content: ""
      - action: INSERT_AFTER
        target:
          snippet: "Line 2 to have nothing inserted after."
        content: ""
      - action: DELETE
        target:
          snippet: "Line 3 to be deleted."
