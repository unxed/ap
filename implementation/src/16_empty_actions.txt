Line 1 to be replaced with nothing.
Line 2 to have nothing inserted after.
Line 3 to be deleted.
