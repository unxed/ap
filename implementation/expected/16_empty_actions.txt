Line 2 to have nothing inserted after.
